src/pages/sign-up/SignUp.scss

```diff
@@ -25,3 +25,7 @@
   @apply text-3xl lg:text-4xl text-start select-none p-5 text-white;
   font-family: 'Lato', sans-serif;
 }
+input::-webkit-inner-spin-button {
```

Comment:
don't we need a .input? Does this affect all input fields? You only need to remove up and down arrow for phone number button. make sure this is not generic and wont affect the whole project (generic one's can affect the whole project . Earlier I had an issue like that)

Reply:
Yes it is nothing but removing the arrow button. let me check if it is affecting any other.